pkg/model/resource/resource.go

```diff
@@ -62,3 +65,19 @@ func (r *Resource) GVK() config.GVK {
 		Kind:    r.Kind,
 	}
 }
+
+func wrapKey(key string) string {
+	return fmt.Sprintf("%%[%s]s", key)
+}
+
+func (r Resource) Replacer() *strings.Replacer {
+	replacements := make([]string, 0)
```

Comment:
`var replacements []string` ?

Reply:
Linters prefer the `make` version, I don't know the exact reasoning